runner.py

```diff
@@ -92,10 +93,18 @@ def run_run(args):
                 raise e
             finally:
                 os.remove(inputFile.name)
-
         else:
             print("All outputs found")
 
+def run_docker_build(args):
```

Comment:
Can we just have bash scripts instead? I don't see the benefit to running commands via python, but it does make it harder to copy/paste/modify them.